Wrappers/Hash.vcxproj

```diff
@@ -12,26 +12,26 @@
   </ItemGroup>
   <PropertyGroup Label="Globals">
     <ProjectGuid>{D8FEE320-54D4-4779-A9C0-F97521ACE759}</ProjectGuid>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
Is there a reason why this is upgraded to only 4.7.2 while the other .net target versions are upgraded to 4.8?

Reply:
Thank you for pointing out the inconsistency in the `.NET` target versions across different projects. I'm not as familiar with `.NET` because standard C++ doesn't rely on it, so I didn't pay close attention to the versioning. VS2022 might have automatically done the upgrade to 4.7.2.

In the future, I'll look into manually upgrading to `.NET 4.8`. Your input is greatly appreciated, and I welcome any further suggestions or concerns you may have.